examples/rust-data-loader/rust/Cargo.toml

```diff
@@ -10,11 +10,10 @@ crate-type = ["cdylib"]
 anyhow = "1.0"
 prost = "0.13"
 serde = { version = "1.0", features = [ "derive" ] }
-foxglove_data_loader = { path = "../../../../sdk/rust/foxglove_data_loader" }
+foxglove_data_loader = { git = "https://github.com/foxglove/foxglove-sdk.git" }
```

Comment:
Should we set this to a specific commit so that if we iterate on the api the example remains valid until we explicitly update it?